lib/html/template/rose-bush/job-entry.html

```diff
@@ -27,6 +27,8 @@
     <!-- entry: status -->
     {% if no_fuzzy_time -%}
       {% set no_fuzzy_time_str = "&no_fuzzy_time=" + no_fuzzy_time -%}
```

Comment:
No, the file has `*.html` extension, and so the viewer thinks that the ampersand is the beginning of an entity that is not terminated by a semicolon. In reality, it is in a Jinja2 quoted string.